src/main/java/edu/kit/kastel/sdq/artemis4j/grading/penalty/Points.java

```diff
@@ -1,4 +1,10 @@
 /* Licensed under EPL-2.0 2024. */
 package edu.kit.kastel.sdq.artemis4j.grading.penalty;
 
-public record Points(double score, boolean capped) {}
+public record Points(double score, boolean capped) {
+    public Points {
+        if (score == -0.0) {
```

Comment:
Is this really necessary? Floats compare as +0.0 == -0.0, no?

Reply:
I don't know. In the old code it returned 0.0, so to keep the behaviour, I added that code.